### PR TITLE
Create nb.json

### PR DESCRIPTION
**Pull request by hwikene**
Translation for Norwegian

### CODE REVIEW

File: custom_components/audiconnect/translations/nb.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "config": {
+        "abort": {
+            "invalid_credentials": "Ugyldige legitimasjon",
```
Comment (Danielhiversen) [inline custom_components/audiconnect/translations/nb.json:4]:
```suggestion
            "invalid_credentials": "Ugyldige innloggingsopplysninger",
```

File: custom_components/audiconnect/translations/nb.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            "user_already_configured": "Kontoen er allerede konfigurert"
+        },
+        "create_entry": {
+        },
+        "error": {
+            "invalid_credentials": "Ugyldige legitimasjon",
```
Comment (Danielhiversen) [inline custom_components/audiconnect/translations/nb.json:10]:
```suggestion
            "invalid_credentials": "Ugyldige innloggingsopplysninger",
```

File: custom_components/audiconnect/translations/nb.json (hunk 1 above)
Comment (hwikene) [reply custom_components/audiconnect/translations/nb.json:4]: I second that 👍

File: custom_components/audiconnect/translations/nb.json (hunk 2 above)
Comment (Danielhiversen) [reply custom_components/audiconnect/translations/nb.json:10]: You did not change this?

**Pull request merged**